openapi-load/src/main/kotlin/pro/bilous/difhub/converter.kt

```diff
@@ -1,12 +1,15 @@
 package pro.bilous.difhub
 
+import pro.bilous.difhub.config.DatasetStatus
+import pro.bilous.difhub.config.SystemSettings
 import pro.bilous.difhub.console.Console
 import pro.bilous.difhub.convert.DifHubToSwaggerConverter
 import pro.bilous.difhub.write.YamlWriter
 
 fun main() {
 	Console.select()
-	DifHubToSwaggerConverter(Console.system!!).convertAll().forEach {
+	val systemSettings = SystemSettings(Console.system!!, DatasetStatus.APPROVED)
```

Comment:
@agentfan please read the status from the file, we are using this function to generate code from command line

Reply:
fixed

Reply:
I investigated the converter.kt and linked files and couldn't find where the file of settings is loaded. It seems like the system with name System.getProperty("DIFHUB_ORG_NAME") or "Demo system" is loaded from "https://metaserviceprod.azurewebsites.net/api". All the Console.select() does is getiing the name of loaded system. So, I desided to set status 'Approved' as default.

Reply:
Entire settings.yaml is set to additionalSettings map. 
Please investigate the following way to run the application.  
<img width="1055" alt="Screenshot 2021-05-13 at 08 33 52" src="https://user-images.githubusercontent.com/19833841/118082548-0c86f600-b3c6-11eb-87bb-a505378dc891.png">

Reply:
Since we load settings only for the code generation, please implement the ability to pass argument to activate Draft loading as well.

<img width="1055" alt="Screenshot 2021-05-13 at 08 33 52" src="https://user-images.githubusercontent.com/19833841/118082770-6daec980-b3c6-11eb-8461-c424e99f9f43.png">

Reply:
added the possibility of selection the datasetStatus from console before startinth the DifHubToSwaggerConverter